docs/merged_objects.md

```diff
@@ -393,3 +393,85 @@ merged_adata_object.var # gene metrics
+The data matrix `raw.X` of the merged `AnnData` object contains the CITE-seq expression data as primary integer counts, and the data matrix `X` contains the RNA-seq expression data as normalized counts.
+Note that only cells which are denoted as `"Keep"` in  the `merged_adata_object.uns["adt_scpca_filter"]` column (as described [above](#singlecellexperiment-cell-metrics)) have normalized expression values in the `X` matrix, and all other cells are assigned `NA` values.
+The data is stored as a sparse matrix, where each column represents a cell or droplet, and each row represents a gene.
+The `raw.X` and `X` matrices can be accessed with the following python code:
+
+```python
+merged_citeseq_adata_object.raw.X # raw count matrix
+merged_citeseq_adata_object.X # normalized count matrix
+```
+
+Column names are cell barcode sequences prefixed with the originating library id, e.g. `SCPCL000000-{barcode}`, and row names are the ADT IDs.
+
+```python
+merged_citeseq_adata_object.obs_names # matrix column names
+merged_citeseq_adata_object.var_names # matrix row names
+```
+
+All of the per-cell data columns included in the `colData` of the `"adt"` alternative experiment in `SingleCellExperiment` merged objects are present in the `.obs` slot of the CITE-seq `AnnData` object.
+To see a full description of the included columns, see the section [on additional `SingleCellExperiment` components for CITE-seq libraries](#additional-singlecellexperiment-components-for-cite-seq-libraries-with-adt-tags).
+
+
+The following QC statistics from [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) can be found in the `obs` slot of each ADT-specific `AnnData` object.
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `zero.ambient`   | Indicates whether the cell has zero ambient contamination   |
+| `sum.controls` |  The sum of counts for all control features. Only present if negative/isotype control ADTs are present |
+| `high.controls`  | Indicates whether the cell has unusually high total control counts. Only present if negative/isotype control ADTs are present |
+| `ambient.scale` |  The relative amount of ambient contamination. Only present if negative control ADTs are _not_ present |
+| `high.ambient`  | Indicates whether the cell has unusually high contamination. Only present if negative/isotype control ADTs are _not_ present |
+| `discard`       | Indicates whether the cell should be discarded based on QC statistics. The `TRUE` and `FALSE` values in this column correspond, respectively, to values `"Discard"` and `"Keep"` in the `adata_obj.obs["adt_scpca_filter"]` column                                                         |
```

Comment:
Aren't these the same as the columns that you mention above as being the same as in the SCE object? I think you either want to include all of the columns or just reference the SCE section and remove this table.

Reply:
> Aren't these the same as the columns that you mention above as being the same as in the SCE object

Yeah, they're the same - _except_ for the reference to the `in the adata_obj.obs["adt_scpca_filter"] column`, which is why I decided to keep it. I can delete it and just write this sentence out as a stand-alone sentence.

One reason (the reason?) this table is here is because it's also in the single SCE file - https://github.com/AlexsLemonade/scpca-docs/blob/development/docs/sce_file_contents.md#additional-anndata-components-for-cite-seq-libraries-with-adt-tags - and there was obviously some copy/pasting here... Do you want to also see it removed there too, in parallel?